AUTHORS.rst

```diff
@@ -24,7 +24,7 @@ Documentation Contributors
 - xCROv `@xCROv <https://github.com/xCROv>`_
 - taq `@greentaquitos <https://github.com/greentaquitos>`_
 
-<!-- - Add "Name <email (optional)> and github profile link" above this line. -->
+.. Add "Name <email (optional)> and github profile link" above this line.
```

Comment:
```suggestion
- Add "Name <email (optional)> and github profile link" above this line.

```
I think it would better to revert to how we had it before. I believe the intention is to have a template there for the contributor to easily copy and change.

Reply:
This actually really makes sense. The way I did it, it would only be seen when editing the file and could cause some confusion.